Formula/r/reprepro.rb

```diff
@@ -36,16 +36,21 @@ class Reprepro < Formula
     cause "No support for GNU C nested functions"
   end
 
+  # Replace strcmp2 with strcmp for consistent string comparison
+  # upstream patch PR, https://salsa.debian.org/debian/reprepro/-/merge_requests/10
+  patch do
+    url "https://raw.githubusercontent.com/Homebrew/formula-patches/5426d605c0d6b6bf4c2e315fed7a991f447caf1a/reprepro/5.4.4-strcmp2.patch"
+    sha256 "cf1aacf08960cb89896a739086d170417820fe894044d0f446b4ff303a1182e5"
+  end
```

Comment:
I don't think the patch is correct; it seems likely to result in UB: https://salsa.debian.org/debian/reprepro/-/blob/21a227f5fb380672ba022d47628895d2f023bde9/globals.h#L97

Reply:
yeah, I thought the same thing, I will debug a bit today.

Reply:
Maybe it's just missing an
```
include "globals.h"
```
somewhere?